subgraph/tests/common.ts

```diff
@@ -22,3 +22,18 @@ export function mockPollContract(): void {
     "getDeployTimeAndDuration():(uint256,uint256)",
   ).returns([ethereum.Value.fromI32(30), ethereum.Value.fromI32(40)]);
 }
+
+export function mockMaciContract(): void {
+  createMockedFunction(
+    // matchstick default event address
+    Address.fromString("0xA16081F360e3847006dB660bae1c6d1b2e17eC2A"),
+    "getPoll",
+    "getPoll(uint256):((address,address,address))",
+  )
+    .withArgs([ethereum.Value.fromUnsignedBigInt(BigInt.fromI32(1))])
+    .returns([
+      ethereum.Value.fromAddress(DEFAULT_POLL_ADDRESS),
+      ethereum.Value.fromAddress(DEFAULT_MESSAGE_PROCESSOR_ADDRESS),
+      ethereum.Value.fromAddress(DEFAULT_TALLY_ADDRESS),
+    ]);
```

Comment:
I think it can be a reason of the error

```suggestion
    .returns(
      ethereum.Value.fromAddress(DEFAULT_POLL_ADDRESS),
      ethereum.Value.fromAddress(DEFAULT_MESSAGE_PROCESSOR_ADDRESS),
      ethereum.Value.fromAddress(DEFAULT_TALLY_ADDRESS),
    );
```

Reply:
No, it's correct way to mock tuples